src/Widgets/FormRatingWidget/IconEditorTab/AllIcons.tsx

```diff
@@ -114,7 +114,7 @@ const Category: React.FC<CategoryProps> = ({
           // instead of creating one SingleIcon component for each of the 675 icons.
           return (
             <div
-              key={`${icon.id}${innerIndex}`}
+              key={`${icon.id}-${category.replaceAll(" ","-")}-${innerIndex}`}
```

Comment:
`key={`${icon.id}-${category.replaceAll(" ","-")}-${innerIndex}`}`

I think you don't need `innerIndex` any more or I am wrong?

Reply:
You are right. icon.id is unique so there is not really a need for the index. 
Removed it.